### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by lorenzwalthert**
Adding a few words on what styler is plus links and defer large example further down in the README. Is that what you had im mind @krlmlr? Not sure how to update pkgdown now though.

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -18,20 +18,38 @@ knitr::opts_chunk$set(
...
+style_text(uggly_code)
+
+```
+
+You can find more information on the wiki of [Google Summer of Code 2017](https://github.com/rstats-gsoc/gsoc2017/wiki/Noninvasive-source-code-formatting) 
+or check out the [pkgdown](https://krlmlr.github.io/styler/) page.
+
+In the following, we present a dirty file and how it is styled with styler. 
+The file can either be tidied, which is a less strict form of styling with an 
+emphasis on preserving the programmers intentions - or cleaned, which means to
```
Comment (krlmlr) [inline README.Rmd]: programmer's

File: README.Rmd [hunk 2]
```diff
@@ -18,20 +18,38 @@ knitr::opts_chunk$set(
...
+You can style a simple character vector of code with `style_text()`, a directory 
+with `style_src()`or a whole R package with `style_pkg()`.
 ```{r echo=FALSE, message=FALSE}
 devtools::load_all()
 #knitr::opts_chunk$set(comment = NA)
 ```
 
 
 ```{r}
+uggly_code <- "a<-function( x){1+1}           "
```
Comment (krlmlr) [inline README.Rmd]: ugly

File: README.Rmd [hunk 3]
```diff
@@ -18,20 +18,38 @@ knitr::opts_chunk$set(
...
-[![Project Status: WIP - Initial development is in progress, but there has not yet been a stable, usable release suitable for the public.](http://www.repostatus.org/badges/latest/wip.svg)](http://www.repostatus.org/#wip)
 [![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/krlmlr/styler?branch=master&svg=true)](https://ci.appveyor.com/project/lorenzwalthert/styler)
+[![Project Status: WIP - Initial development is in progress, but there has not yet been a stable, usable release suitable for the public.](http://www.repostatus.org/badges/latest/wip.svg)](http://www.repostatus.org/#wip)
 
-The goal of styler is to provide non-invasive pretty-printing of R source code while adhering to general formatting conventions.
-
-## Dirty source file
+The goal of styler is to provide non-invasive pretty-printing of R source code 
+while adhering to the [tidyverse](https://github.com/tidyverse/style) formatting 
+rules. Custom style guides might also be supported in the future.
```
Comment (krlmlr) [inline README.Rmd]: Support for custom style guides is planned.

**Comment by lorenzwalthert** [comment]
ok, thanks. I think we can also wait with another release until we have accumulated some more changes.

**Pull request merged**